android/timodule.xml

```diff
@@ -1,20 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:module xmlns:ti="http://ti.appcelerator.org" xmlns:android="http://schemas.android.com/apk/res/android">
 	<!--
-		Similar to tiapp.xml, but contains module/platform specific
-		configuration in <iphone>, <android>, and <mobileweb> sections
+	Similar to tiapp.xml, but contains module/platform specific
+	configuration in <iphone>, <android>, and <mobileweb> sections
 	-->
-	<iphone>
-	</iphone>
+	<iphone></iphone>
 	<android xmlns:android="http://schemas.android.com/apk/res/android">
 		<manifest>
 			<uses-permission android:name="android.permission.BLUETOOTH"/>
 			<uses-permission android:name="android.permission.BLUETOOTH_ADMIN"/>
 			<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"></uses-permission>
 			<uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"></uses-permission>
 			<uses-feature android:name="android.hardware.bluetooth_le" android:required="true"/>
+			<application>
+				<receiver
+				android:exported="true"
+				android:name=".ti.bluetooth.BlutoothStateChangedBroadcastReceiver"
+				android:enabled="true">
+					<intent-filter>
+						<action android:name="android.bluetooth.adapter.action.STATE_CHANGED" />
+						<action android:name="android.bluetooth.device.action.BOND_STATE_CHANGED" />
+						<action android:name="android.bluetooth.a2dp.profile.action.CONNECTION_STATE_CHANGED" />
+					</intent-filter>
+				</receiver>
+			</application>
 		</manifest>
 	</android>
-	<mobileweb>
-	</mobileweb>
+	<mobileweb></mobileweb>
 </ti:module>
```

Comment:
Can you put the changes of this file in an own PR? Then we could merge it directly.